app/src/main/java/org/oppia/android/app/settings/profile/ProfileEditFragmentPresenter.kt

```diff
@@ -151,16 +149,11 @@ class ProfileEditFragmentPresenter @Inject constructor(
         fragment,
         Observer {
           if (it is AsyncResult.Success) {
-            if (fragment.requireContext().resources.getBoolean(R.bool.isTablet)) {
-              val intent =
-                Intent(fragment.requireContext(), AdministratorControlsActivity::class.java)
-              intent.addFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP)
-              fragment.startActivity(intent)
-            } else {
-              val intent = Intent(fragment.requireContext(), ProfileListActivity::class.java)
-              intent.addFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP)
-              fragment.startActivity(intent)
-            }
+            ProfileDeleteSuccessDialogFragment.createNewInstance()
+              .showNow(
+                fragment.childFragmentManager,
+                ProfileDeleteSuccessDialogFragment.DELETE_PROFILE_SUCCESS_DIALOG_FRAGMENT_TAG
+              )
```

Comment:
Here is an example of a test that verifies a dialog is shown -> https://github.com/oppia/oppia-android/blob/2f68639f90e84fde029b92da48a9bc925cea7cc8/app/src/sharedTest/java/org/oppia/android/app/devoptions/MarkChaptersCompletedFragmentTest.kt#L706. Please add a test for this change in `ProfileEditFragmentTest`.